client/src/featureform/register.py

```diff
@@ -1276,34 +1277,22 @@ def __init__(self, host=None, local=False, insecure=True, cert_path=None):
         """
         super().__init__()
         self.local = local
-        if self.local:
+        if local:
             if host != None:
                 raise ValueError("Cannot be local and have a host")
         else:
-            if host == None:
-                host = os.getenv('FEATUREFORM_HOST')
-                if host == None:
-                    raise ValueError(
-                    "Host value must be set or in env as FEATUREFORM_HOST")
-
-            channel = self.tls_check(host, cert_path, insecure)
+            host = host or os.getenv('FEATUREFORM_HOST')
```

Comment:
if host and `FEATUREFORM_HOST` are both set, would we use `host` instead of the env?